### PR TITLE
QRDA Import adds and fixes

### PR DESCRIPTION
**Pull request by sjpadgett**
CCDA 2.16.840.1.113883.10.20.22.4.13 - Procedure Activity Observation.
QRDA 2.16.840.1.113883.10.20.24.3.18 - Diagnostic Study, Performed,
QRDA 2.16.840.1.113883.10.20.24.3.59 - Physical Exam, Performed

### CODE REVIEW

**Comment by sjpadgett** [comment]
@kchapple I'm leaving open during testing. I sent you a patch to work with...

File: sql/6_0_0-to-6_1_0_upgrade.sql [hunk 1]
```diff
@@ -1261,3 +1261,25 @@ UPDATE `layout_options` SET `codes` = 'SNOMED-CT:67822003' WHERE `form_id` = 'HI
...
+ALTER TABLE `form_observation` ADD `result_status` VARCHAR(32) NULL;
+#EndIf
+#IfMissingColumn form_observation ob_reason_status
+ALTER TABLE `form_observation` ADD `ob_reason_status` VARCHAR(32) NULL;
+#EndIf
+#IfMissingColumn form_observation ob_reason_code
+ALTER TABLE `form_observation` ADD `ob_reason_code` VARCHAR(32) NULL;
+#EndIf
+#IfMissingColumn form_observation ob_reason_text
+ALTER TABLE `form_observation` ADD `ob_reason_text` TEXT NULL;
```
Comment (bradymiller) [inline sql/6_0_0-to-6_1_0_upgrade.sql:1278]: I received a message from the text null police:
"remove the NULL here" :)

File: sql/database.sql [hunk 2]
```diff
@@ -11421,7 +11421,14 @@ CREATE TABLE `form_observation` (
   `code_type` varchar(255),
   `table_code` varchar(255),
   `ob_code` VARCHAR(31) DEFAULT NULL,
-  `ob_type` VARCHAR(31) DEFAULT NULL
+  `ob_type` VARCHAR(31) DEFAULT NULL,
+  `ob_status` varchar(32) DEFAULT NULL,
+  `result_status` varchar(32) DEFAULT NULL,
+  `ob_reason_status` varchar(32) DEFAULT NULL,
+  `ob_reason_code` varchar(255) DEFAULT NULL,
+  `ob_reason_text` text DEFAULT NULL,
```
Comment (bradymiller) [inline sql/database.sql:11429]: I received a message from the text null police:
"remove the DEFAULT NULL here" :)

Also, for the other entries, I don't know the difference between 'DEFAULT NULL' or 'NULL' (and there may not be a differebce), but would choose one (NULL is used in upgrade and DEFAULT NULL is used here in database.sql)

**Comment by bradymiller** [comment]
2DV

File: sql/database.sql (hunk 2 above)
Comment (sjpadgett) [reply sql/database.sql:11429]: yep I keep forgetting:) doesn't make a diff but for consistency then.. thanks

**Pull request merged**